carma_cooperative_perception/src/multiple_object_tracker_component.cpp

```diff
@@ -482,10 +569,14 @@ auto MultipleObjectTrackerNode::execute_pipeline() -> void
   temporally_align_detections(detections_, current_time);
 
   const auto predicted_tracks{predict_track_states(track_manager_.get_all_tracks(), current_time)};
-  const auto scores{
-    mot::score_tracks_and_detections(predicted_tracks, detections_, mot::euclidean_distance_score)};
+  auto scores{
+    mot::score_tracks_and_detections(predicted_tracks, detections_, SemanticDistance2dScore{})};
+
+  mot::prune_track_and_detection_scores_if(scores, [](const auto & score) { return score > 5.0; });
```

Comment:
Why does the class use 5.0? Could we add explanation or comment

Reply:
Done